src/test/suite/fixtures/recorded/languages/latex/changeArg.yml

```diff
@@ -0,0 +1,27 @@
+languageId: latex
```

Comment:
Would be good to support / test "chuck arg" as well, which should remove the curly braces